alphanet-walletextension-sdk/lib/__tests__/transactions.spec.ts

```diff
@@ -0,0 +1,194 @@
+/* eslint-disable array-callback-return */
+/* eslint-disable max-nested-callbacks */
+
+import {
+  Bool,
+  Box,
+  Bucket,
+  CollectionType,
+  ComponentAddress,
+  Decimal,
+  Enum,
+  Hash,
+  I128,
+  I16,
+  I32,
+  I64,
+  I8,
+  List,
+  Map,
+  NonFungibleAddress,
+  Option,
+  PackageAddress,
+  PreciseDecimal,
+  ResourceAddress,
+  Result,
+  Set,
+  String,
+  TransactionSpecError,
+  Tuple,
+  BasicType,
+  U128,
+  U16,
+  U32,
+  U64,
+  U8,
+  Unit,
+  Vec,
+  Expression,
+} from '../transaction-spec'
+
+describe('transation spec', () => {
+  it.each([
+    [Unit(), '()'],
+    [Bool(false), 'false'],
+    [Bool(true), 'true'],
+    [I8(8), '8i8'],
+    [I16(16), '16i16'],
+    [I32(32), '32i32'],
+    [I64('64'), '64i64'],
+    [I128('128'), '128i128'],
+    [U8(8), '8u8'],
+    [U16(16), '16u16'],
+    [U32(32), '32u32'],
+    [U64('64'), '64u64'],
+    [U128('128'), '128u128'],
+    [Decimal(42.42), 'Decimal("42.42")'],
+    [PreciseDecimal(42.42), 'PreciseDecimal("42.42")'],
+    [Expression('ENTIRE_WORKTOP'), 'Expression("ENTIRE_WORKTOP")'],
+    [Enum('Foo'), 'Enum("Foo")'],
+    [
+      Enum('Foo', String('some string'), Bool(false)),
+      'Enum("Foo","some string",false)',
+    ],
+    [Option.Some(String('foobar')), 'Some("foobar")'],
+    [Option.Some(U128('128')), 'Some(128u128)'],
+    [Option.Some(Decimal(219.01234)), 'Some(Decimal("219.01234"))'],
+    [Option.None, 'None'],
+    [Box(U64('64')), 'Box(64u64)'],
+    [Box(Decimal(42.42)), 'Box(Decimal("42.42"))'],
```

Comment:
Seem to be using old spec.

You can find the Scrypto v0.6.0 spec [here](https://github.com/radixdlt/radixdlt-scrypto/blob/v0.6.0/sbor/src/any.rs).